UI.AutomationTests/Video/AddParticipantsPostBookingTests.cs

```diff
@@ -52,11 +52,6 @@ await TestContext.Out.WriteLineAsync(
         });
 
         var judgeHearingRoomPage = judgeWaitingRoomPage.StartOrResumeHearing();
-        foreach (var participant in participantsToAdd)
-        {
-            var p = participantsInConference.First(x => x.Username == participant.Username);
-            judgeHearingRoomPage.AdmitParticipant(p.DisplayName, p.Id.ToString());
```

Comment:
What's replacing this admin-participants part?

Reply:
you don't need it if the participant role is not a witness. especially since they participant is being added before a hearing is started